ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -52,6 +52,7 @@
         - (ansible_distribution == "RedHat" or ansible_distribution == "CentOS" or ansible_distribution == "Ubuntu" or ansible_distribution == "SLES")
         - ansible_architecture == "x86_64"
     - OpenSSL102                  # OpenJ9
+    - Python3_Source
```

Comment:
You might remove Python3_Source role as this is a requirement for an OpenJ9 future test.

Reply:
The role has been removed